hugegraph-mysql/src/main/java/com/baidu/hugegraph/backend/store/mysql/MysqlSerializer.java

```diff
@@ -128,7 +128,7 @@ protected void formatProperties(HugeElement element,
     protected void parseProperties(HugeElement element,
                                    TableBackendEntry.Row row) {
         String properties = row.column(HugeKeys.PROPERTIES);
-        // Query edge will wraped by a vertex, whose properties is empty
+        // Query edge will wrapped by a vertex, whose properties is empty
```

Comment:
will wrapped --> `wil be wrapped`?    (Can this be detected? lack `be`)

Reply:
Kinda. It isn't a grammar checker, but if you know someone is going to make a certain kind of mistake and can describe it with a regular expression, then, yes.

This is the set of supplemental rules that were used for this PR:
https://github.com/jsoref/hugegraph/blob/spell-check/.github/actions/spelling/line_forbidden.patterns

Only the last one is actually fancy.

But, you could do something like:
```
# s.b. will _be_ ...
\bwill [a-z]{3,}ed\b
```

Reply:
Note that this heuristic would fail for `will speed up` (bumping to `{4,}` would covert that case).

I haven't thought through these edges before.

In theory, there may be grammar checkers available for GitHub, although, I can't point to any as GitHub Actions.